packages/serverless/src/awslambda.ts

```diff
@@ -66,7 +76,7 @@ interface AWSLambdaOptions extends Sentry.NodeOptions {
 }
 
 /**
- * @see {@link Sentry.init}
+ * @see {@link init}
```

Comment:
l: I don't think actually links to anywhere, right? We might just add a one-liner JSDoc instead, wdyt?

```suggestion
 * Initializes the Sentry AWS Lambda SDK.
 * 
 * @param options Configuration options for the SDK, @see {@link AWSLambdaOptions}.
```